env

```diff
@@ -33,8 +33,10 @@ TWITTER_SECRET=
 #   For the callback URL use 'http://<your hostname:port>/auth/google/callback'
 #   Once registered copy the ID and Secret here
 #
+#   GOOGLE_OAUTH2_HD: (Optional) Limit sign-in to a particular Google Apps hosted domain. This can be simply string 'domain.com'
```

Comment:
Line 36 should follow same format as rest of the `env` file.

#   The GOOGLE_OAUTH2_HD variable is used to limit sign-in to a particular Google Apps hosted domain. This
#    can be a string such as, 'domain.com'. If left blank, GreenLight will allow sign-in from all Google Apps hosted
#    domains.
#